### PR TITLE
migrate examples to openApi part 17; affects [buildkite cii codeship crates jsdelivr npms] and bundlephobia

### PR DESCRIPTION
**Pull request by chris48s**
Refs https://github.com/badges/shields/issues/9285

I have touched fewer files in this PR as I'm starting to get into some more slightly more involved services now.

I'll be honest. 17 PRs deep into this project and knowing how many more there are to do, my enthusiasm for this endeavour is starting to decrease a bit. Staying strong though.

I've skipped the bundlephobia tests on this PR. They seem to be having some kind of outage at the moment affecting some packages but not others e.g:

https://bundlephobia.com/api/size?package=react (works)
https://bundlephobia.com/api/size?package=preact (error)

lets keep an eye on that one..

### CODE REVIEW

File: services/crates/crates-downloads.service.js [hunk 1]
```diff
@@ -9,49 +9,54 @@ export default class CratesDownloads extends BaseCratesService {
     pattern: ':variant(d|dv|dr)/:crate/:version?',
```
Comment (PyvesB) [inline services/crates/crates-downloads.service.js:9]: For the jsdelivr badge you leveraged a period enum, whereas here you broke down wvery example. Any specific reason why?

Comment (chris48s) [reply services/crates/crates-downloads.service.js:9]: Yes. I think the variants here need more explanation.

I don't think I would know

d = total downloads
dv = downloads for version
dr = recent downloads

so I think it makes sense to give each one an individual title.

In the jsdelivr example, it was more straightforward to just explain this in the param description: `description: 'Hits per day, week, month or year',`.

Comment (PyvesB) [reply services/crates/crates-downloads.service.js:9]: Thanks for explaining, makes sense to me.

**Review by PyvesB — APPROVED** [review]
Looks good!

**Pull request merged**